e2e/bzlmod/BUILD

```diff
@@ -15,8 +15,8 @@ proto_library(
 
 ts_proto_library(
     name = "foo_ts_proto",
-    has_services = False,
     copy_files = False,
+    gen_connect_es = False,
```

Comment:
normally we couldn't land a breaking change on `main` - but the top of `proto.md` claims that this rule isn't yet a stable public API so we can still get away with it here :)